BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTestsBase.cs

```diff
@@ -8,5 +8,6 @@ namespace BricksAndHearts.UnitTests.ControllerTests.Landlord;
 public class LandlordControllerTestsBase : ControllerTestsBase
 {
     public static readonly IPropertyService propertyService = A.Fake<IPropertyService>();
+    public static readonly ILandlordService landlordService = A.Fake<Services.LandlordService>();
     protected readonly LandlordController _underTest = new(null!, null!, new LandlordService(null!), propertyService);

```

Comment:
We shouldn't be using static fakes, they should be fresh for each test. Can you add a constructor and set these up in there instead? Then you can use them in the test rather than creating them each time

Reply:
I deleted these lines and all the tests still pass, , so it seems as though they're not required.
![image](https://user-images.githubusercontent.com/36637730/181475460-ccf15d8c-ea80-4e4c-a2b7-4fe691037bb2.png)

Reply:
All that was needed was to generate the fake in the UnderTest controller

Reply:
I've just seen there's more in the other files too :S

Reply:
I've rebased onto main because it already contained this fix